lib/metrics/processRequests.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var Gauge = require('../gauge');
+
+module.exports = function() {
+    var gauge = new Gauge('node_active_handles_total', 'Number of active requests.');
+
+    return function() {
+        gauge.set(process._getActiveRequests().length);
```

Comment:
It's the current recommended way. https://github.com/nodejs/node/issues/1128#issuecomment-78409928